OpenOversight/app/utils/general.py

```diff
@@ -1,7 +1,6 @@
 import random
 import sys
 from collections.abc import Callable, Hashable, Iterable
-from distutils.util import strtobool
```

Comment:
`distutils` was deprecated in Python 3.10 and removed in Python 3.12. Since we're using Python 3.12 for testing, `distutils.util.strtobool` had to be replaced.